translate/convert/test_po2moz.py

```diff
@@ -1,8 +1,70 @@
-from translate.convert import po2moz, test_convert
+from translate.convert import po2moz, po2prop, test_convert
+from translate.misc import wStringIO
+from translate.storage import po
 
 
 class TestPO2Moz(object):
-    pass
+
+    def merge2prop(self, propsource, posource, personality="java",
+                   encoding='utf-8'):
+        """Helper that merges po translations to .properties source without requiring files"""
+        inputfile = wStringIO.StringIO(posource)
+        templatefile = wStringIO.StringIO(propsource)
+        outputfile = wStringIO.StringIO("")
+        po2prop.convertmozillaprop(inputfile, outputfile, templatefile)
+        print(outputfile.getvalue())
+        return outputfile.getvalue()
+
+    def test_properties_duplicates(self):
+        """Test back conversion of properties with duplicate units."""
+        proptemplate = '''
```

Comment:
This looks like a complete copy over of po2prop.  po2moz is just a wrapper.  Better to fix this properly in po2prop.  It also isn't addressing the test that passed that should have failed.